CodeStarterCamp_Week4/main.swift

```diff
@@ -1,12 +1,21 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week4
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
-print("Hello, World!")
+struct Routine {
+    let name: String
+    var exercises: Array<Exercise>
+    
+    func start() {
+        print("hellRoutine을 시작합니다.")
+        for exercise in exercises {
+            exercise.action()
+            print(exercise.name)
+        }
+    }
+}
+
+var player = BodyCondtion(upperBodyStrength: 0, lowerBodyStrength: 0, muscleEndurance: 0, tiredness: 0)
+let exercises : Array<Exercise> = [sitUP, sitUP, activeRest, Squatting, Squatting, activeRest, longRun, longRun]
+var hellRoutine = Routine(name: "hellRoutine", exercises: exercises)
```

Comment:
- 위에서 BodyCondition 타입의 이니셜라이저를 하나 더 만들었다면 (편리한 형태로)
여기서는 아래와 같이 간단하게 player 인스턴스를 생성할 수 있습니다.
그리고 var → `let` 으로 받아도 문제가 없어요! (클래스는 레퍼런스 타입!)

- exercises 배열이 다소 길기 때문에, 외부에서 배열 만든 다음에 파라미터로 넣어주는 거 좋아요! 😄 
궁금한 점은, 바둑이님은 배열을 표현할 때, 왜 `[Exercise]` 보다 `Array<Exercise>` 표현을 좋아하세요~?
어떤 걸 사용하시든 괜찮지만, 보통은 `대괄호`를 사용하는 게 일반적이거든요!

- hellRoutine 인스턴스도 var → `let`으로 받아도 문제가 없습니다.
항상 let 을 먼저 생각하시고, 변경해야 할 필요가 있는 경우에만 var 로 바꿔주시면 좋아요!

```suggestion
let player = BodyCondition()
let exercises: [Exercise] = [sitUP, sitUP, activeRest, Squatting, Squatting, activeRest, longRun, longRun]
let hellRoutine = Routine(name: "hellRoutine", exercises: exercises)
```